docs/current/architecture.md

```diff
@@ -22,68 +22,74 @@
 
 **Last Updated**: February 2017
 
-Vaquero is designed to simplify the provisioning and ongoing operations of clustered software on bare metal infrastructure. A running Vaquero system is composed of a centralized control plane that automates provisioning of software in one or more data centers, and remote agents that take actions on boot hosts.
+## Architecture
 
-The goal is to provide the ability for teams to manage their infrastructure using the same tools they use for their applications (revision control, CI/CD pipeline, etc.), to and enable similar workflows including automated updates, gating, immutability, and A/B deployments. The final outcome will be a fully operational data center running heterogeneous deployments of clustered software on bare metal, with fully automated deployment and upgrades driven by a CI pipeline.
+For simplicity, Vaquero is delivered as a single docker image with all of the required services available (server and agent). The same docker image can be used to start containers in both the control plane and remote agent sites.
 
-## Architecture
+![](https://ciscocloud.github.io/vaquero-docs/docs/current/jan17Arch.png)
+
+### Terminology
 
-Vaquero is delivered in one container for operational simplicity. Vaquero can run in multiple modes, including server, agent, and standalone (the combination of server and agent).
+The following terms are used throughout the documentation:
 
-![](jan17Arch.png)
+* `source of truth (SoT)`: all configuration necssary to define your infrastructure and available provisioning details (workflows, operating systems, boot configuration, unattended install scripts, and site/host-level information)
 
+### Vaquero Server
 
-#### `vaquero server`
+Vaquero server is the central service that maintains the configuration and workflows for provisioning your hardware. It is the service that users and Vaquero agents interact with. Vaquero server consists of the following components:
```

Comment:
maybe mention inventory/configuration. `Vaquero server is the central service that maintains the inventory and configuration workflows for provisioning your hardware`

Reply:
I like that, added.